aircraft/F-15/Models/f15c.xml

```diff
@@ -120,7 +120,7 @@
                 loop();
             }
             
-            var livery_update = aircraft.livery_update.new("Aircraft/F-15/Models/Liveries", 10 );
+            var livery_update = aircraft.livery_update.new("Aircraft/F-15/Models/liveries", 10 );
```

Comment:
The liveries folder should be upper case - it is github that is wrong; see FGAddon